libs/blocks/merch-card/merch-card.js

```diff
@@ -371,11 +273,15 @@ const init = async (el) => {
   if (el.dataset.removedManifestId) {
     merchCard.dataset.removedManifestId = el.dataset.removedManifestId;
   }
-  if (name) {
-    merchCard.setAttribute('name', name);
-  }
   let tags = {};
-  if (el.lastElementChild) {
+  if (el.lastElementChild?.previousElementSibling?.querySelector('h3,h4,h5,h6')) {
```

Comment:
am i right to assume you aim at heading in the last section of the card that is the one for the tags? doing this we enforce that:
- only one heading should be in that section,
- no other section that could hold headings should be added after the tag section
right?

Reply:
If a heading is present in the first section of the block we ignore it.
>only one heading should be in that section:

we pick the first heading in the last section. But the others are ignored but do not break the feature.

>no other section that could hold headings should be added after the tag section
right?

yes.